cfme/tests/containers/test_relationships_tables.py

```diff
@@ -33,17 +31,20 @@
                           'Containers',
                           'Node'])
 def test_pods_rel(provider, rel):
+    """   This module verifies the integrity of the Relationships table
+          We also verify that clicking on the Relationships table field
+          takes the user to the correct page, and the number of rows
+          that appears on that page is equal to the number in the
+          Relationships table
+    """
     sel.force_navigate('containers_pods')
     list_tbl_pod = CheckboxTable(table_locator="//div[@id='list_grid']//table")
     ui_pods = [r.name.text for r in list_tbl_pod.rows()]
-    mgmt_objs = provider.mgmt.list_container_group()  # run only if table is not empty
-
-    if ui_pods:
-        # verify that mgmt pods exist in ui listed pods
-        assert set(ui_pods).issubset(
-            [obj.name for obj in mgmt_objs]), 'Missing objects'
+    ui_pods_revised = filter(
```

Comment:
Also. I am curious why here it is not OK to use the ``provider.mgmt.list_container_group()`` while on other places it is ok to use the ``provider.mgmt.*``